<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SrsPage</class>
   <item>
   <layout class="QHBoxLayout" name="topInfoLayout">
   <item>
      <widget class="QLabel" name="labelProgress">
      <property name="text">
      <string>0 / 0</string>
      </property>
      </widget>
   </item>

   <item>
      <widget class="QProgressBar" name="progressBar">
      <property name="value">
      <number>0</number>
      </property>
      </widget>
   </item>
   </layout>
   </item>

 <widget class="QWidget" name="SrsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>237</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>16</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
font-size: 22px;
font-weight: bold;
      </string>
   </property>
   <property name="text">
    <string>Modo Repaso</string>
   </property>
  </widget>
  <widget class="QFrame" name="cardFrame">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>58</y>
     <width>176</width>
     <height>120</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="cardLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="margin" stdset="0">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="labelWord">
      <property name="styleSheet">
       <string>
font-size: 28px;
font-weight: bold;
         </string>
      </property>
      <property name="text">
       <string>Palabra</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnShowAnswer">
      <property name="text">
       <string>Show Answer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAnswer">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Significado aqu√≠</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>343</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAgain">
      <property name="text">
       <string>Again</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGood">
      <property name="text">
       <string>Good</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHard">
      <property name="text">
       <string>Hard</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEasy">
      <property name="text">
       <string>Easy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
